disperser/dataapi/server.go

```diff
@@ -58,8 +58,10 @@ type (
 	}
 
 	Metric struct {
-		Throughput          float64                  `json:"throughput"`
-		CostInGas           float64                  `json:"cost_in_gas"`
+		Throughput float64 `json:"throughput"`
+		CostInGas  float64 `json:"cost_in_gas"`
+		// deprecated: use TotalStakePerQuorum instead. Remove when the frontend is updated.
+		TotalStake          uint64                   `json:"total_stake"`
 		TotalStakePerQuorum map[core.QuorumID]uint64 `json:"total_stake_per_quorum"`
```

Comment:
Isn't it already returning by quorum?

Reply:
Not sure if i understand what you mean, but it was only returning the total stake of quorum 0 in `TotalStake` field. Now `TotalStakeByQuorum` is added to return total stakes for all quorums

Reply:
Ah I was reading the second commit